app/controllers/charge.controller.js

```diff
@@ -35,8 +35,29 @@ const { views, preserveProperties } = require('../../config/charge.controller')
 const { CORRELATION_HEADER } = require('../../config/correlation-header')
 const { createChargeIdSessionKey } = require('../utils/session')
 const { getLoggingFields } = require('../utils/logging-fields-helper')
+const applePayEnabled = (process.env.APPLE_PAY_ENABLED || 'true') === 'true'
```

Comment:
Because this is an environment var it's considered a string, `=== true` ensures that we coalesce to a boolean correctly in all circumstances. `|| 'true'` just ensures that the value is defaulted to true in the event that this value hasn't been set properly in the environment.

Reply:
```
> console.log(process.env.EXAMPLE)
true
undefined
> typeof(process.env.EXAMPLE)
'string'
> let myExampleEnvVar = process.env.EXAMPLE
undefined
> typeof myExampleEnvVar
'string'
> myExampleEnvVar = process.env.EXAMPLE === 'true'
true
> typeof(myExampleEnvVar)
'boolean'
```

Reply:
IMO we shouldn't be using environment variables for this kind of thing, something like [node-config](https://github.com/node-config/node-config) would be much better suited!

Reply:
Weird. Not sure what's up with my node shell then.
```
> var exampleVar = process.env.APPLE_PAY_ENABLED || 'true' === 'true'
undefined
> typeof(exampleVar)
'string'
```

Reply:
Let's leave this. No need to fix this comment.